pages/index.vue

```diff
@@ -5,23 +5,38 @@
       header="Best zeppelins"
       subheader="Lowest prices"
       :background-image="backgroundImage"
-      :cta="{ title: 'Check out our flying boys', href: '#', isButton: true }"
+      :cta="{
+        title: 'Check out our flying boys',
+        href: '#',
+        variant: 'fill',
+        arrow: 'right',
+      }"
       class="col-span-2 h-56"
     />
     <RedirectionBanner
       link="/"
       header="Company with decades of history"
       :background-image="backgroundImage"
       transparent-background
-      :cta="{ title: 'Read more', href: '#' }"
+      :cta="{
+        title: 'Read more',
+        href: '#',
+        variant: 'text',
+        arrow: 'right',
+      }"
       class="col-span-1 h-56"
     />
     <RedirectionBanner
       link="/"
       header="Need help?"
       :background-image="backgroundImage"
       transparent-background
-      :cta="{ title: 'Contact us', href: '#' }"
+      :cta="{
+        title: 'Read more',
+        href: '#',
+        variant: 'text',
+        arrow: 'right',
+      }"
```

Comment:
`cta` type has required property of `to` type `string` but you add none here. It throws TS error in my editor. Maybe we should make `to` and `href` not required? 🤔